app/factory/Layer.js

```diff
@@ -880,6 +880,30 @@ Ext.define('CpsiMapview.factory.Layer', {
         return olSourceProps;
     },
 
+    /**
+     * Create a point style for clustered features
+     * @param {any} featCount the number of features in the clustered feature
+     */
+    createClusterStyle: function (featCount) {
```

Comment:
Wouldn't this be placed better in `CpsiMapview.util.Style`?

Reply:
Thanks for the review @chrismayer - yes it would make more sense to move this to the style util class